sql/wsrep_mysqld.h

```diff
@@ -134,6 +135,9 @@ enum enum_wsrep_ignore_apply_error {
 
 extern const char *wsrep_fragment_units[];
 extern const char *wsrep_SR_store_types[];
+enum enum_wsrep_mode {
+    WSREP_MODE_IGNORE_NATIVE_REPLICATION_FILTER_RULES = 0
+};
```

Comment:
Running clang-format produces a different alignment. Please run clang-format on all source files touched by this commit.
```
+enum enum_wsrep_mode { WSREP_MODE_IGNORE_NATIVE_REPLICATION_FILTER_RULES = 0 };
```